prow/config/external_plugins_config.yaml

```
@@ -1575,6 +1575,20 @@ ti-community-format-checker:
           <sub>:open_book: For more info, you can check the ["Contribute Code"](https://pingcap.github.io/tidb-dev-guide/contribute-to-tidb/contribute-code.html#referring-to-an-issue) section in the development guide.</sub>
         missing_label: do-not-merge/needs-linked-issue
         skip_label: skip-issue-check
+  - repos:
+      - pingcap/tiflash
+    required_match_rules:
+      - pull_request: true
+        body: true
+        branches:
+          - planner_refactory
```

```suggestion
          - master
          - planner_refactory
```

And please remove line 1551.